### DISCUSSION TITLE
Automatically save diagrams as SVG?

### DISCUSSION THREAD

**Discussion by lizzygit** [EvotecIT/PSWriteHTML#271] [Q&A]
hey, 
is there any way to automatically save the diagrams as SVG when generating the HTML file? The necessary links are unfortunately dynamically generated, so you can't script this after the HTML is already created. Or is it possible to directly send out the Diagramms as Email?
Thanks

**Comment by PrzemyslawKlys** [comment]
That's not going to work. There's no built-in way to do this. SVG is generated by the browser so you would need to "open" the browser and simulate mouse clicks and so on. Doable with the AutoIt PowerShell module, but far from fun. 

You could use https://github.com/KevinMarquette/PSGraph to generate images, and attach that.
